cmake/globalmacros.cmake

```diff
@@ -538,7 +538,15 @@ function(ivw_create_module)
 
     # Optimize compilation with pre-compilied headers
     if(NOT ARG_NO_PCH)
-        ivw_compile_optimize_on_target(${${mod}_target})
+        if(IVW_PRECOMPILED_HEADERS_MODE STREQUAL "Reuse")
```

Comment:
this should be inside of ivw_compile_optimize_on_target

Reply:
done